### DISCUSSION TITLE
When will the Seaport protocol be used on testnets.opensea.io?

### DISCUSSION THREAD

**Discussion by JasperAlexander** [ProjectOpenSea/seaport#557] [Q&A] [answered]
When will the Seaport protocol be used on testnets.opensea.io? Around two weeks ago I saw that the Seaport protocol was already being used, but now it uses the Wyvern protocol again.

**Comment by Slokh** [comment, marked as answer]
Testnets should be fine for your own contract. Wyvern is only used for the OpenSea Shared Storefront, you can find more context why here: https://twitter.com/z0age/status/1537276889504415744

We're working on a solution for bringing this to Seaport too.
